packages/xrpl/src/models/methods/accountTx.ts

```diff
@@ -49,7 +54,7 @@ export interface AccountTxRequest extends BaseRequest, LookupByLedgerRequest {
   marker?: unknown
 }
 
-export interface AccountTxTransaction {
+export interface AccountTxTransaction<Version extends APIVersion> {
   /** The ledger index of the ledger version that included this transaction. */
   ledger_index: number
   /**
```

Comment:
`ledger_hash` should be present in both the request and response of `AccountTx`.

Here's an example usage in the response (from cpp unit tests) - https://github.com/XRPLF/rippled/blob/c706926ee3c7d0f3abbbd3545cc85d6886643444/src/test/rpc/AccountTx_test.cpp#L162

here's is the usage inside a request: https://github.com/XRPLF/rippled/blob/c706926ee3c7d0f3abbbd3545cc85d6886643444/src/test/rpc/AccountTx_test.cpp#L316

I think the JS client library didn't have `ledger_hash` field since before this PR

Reply:
I just added it in `ResponseOnlyTxInfo`.